experimental-examples/tina-cloud-starter/.tina/schema.ts

```diff
@@ -251,6 +255,7 @@ const schema = defineSchema({
       label: "Global",
       name: "global",
       path: "content/global",
+      ui: true,
```

Comment:
@jeffsee55 Getting a type error here. Is this supposed to be 

```
ui: {
    global: true
},
``` 
?

Reply:
Oh woops, yeah fixed.